src/utils.js

```diff
@@ -0,0 +1,21 @@
+function printWarning(format, ...args) {
+  var argIndex = 0;
+  var message = 'Warning: ' + format.replace(/%s/g, () => args[argIndex++]);
+  if (typeof console !== 'undefined') {
+    /* eslint-disable no-console */
+    console.error(message);
+    /* eslint-enable no-console */
+  }
+}
+
+export function warning(condition, format, ...args) {
```

Comment:
Do we need [all of the it](https://unpkg.com/warning@3.0.0/browser.js)? This one is smaller.

Reply:
it all disappears in production, I think we'll probably get better savings anyway since that's what a lot of libraries and apps use, the likelihood that it's already in a bundle is pretty high, so it'd be free in that case.